src/pages/Academics/AcademicsSubNav.tsx

```diff
@@ -7,10 +7,10 @@ import { Routes } from '@osu-wams/utils';
 const AcademicSubNav = () => {
   return (
     <SubNav>
-      <SubNavLink to={Routes.Routes().academics.fullPath}>
+      <SubNavLink end to={Routes.Routes().academics.fullPath}>
```

Comment:
makes it so that for a CSS active link style to work, the route must match exactly